app/lib/frontend/templates/listing.dart

```diff
@@ -122,12 +122,13 @@ String renderPkgIndexPage(
   final values = {
     'sort_control_html': renderSortControl(searchQuery),
     'is_search': isSearch,
-    'title': title ?? topPackages,
     'package_list_html': renderPackageList(packages, searchQuery: searchQuery),
     'has_packages': packages.isNotEmpty,
     'pagination': renderPagination(links),
-    'search_query': searchQuery?.query,
     'total_count': totalCount,
+    // TODO: remove the keys below after we have migrated to the new design
```

Comment:
Would it make sense to mark these with the issue number to make these TODO's easier to find?
```
TODO(3246): remove the keys below after we have migrated to the new design
```

Reply:
Done. I'll update the earlier similar TODOs if I touch the code nearby.